src/pages/leeyunseung/Main/Main.js

```diff
@@ -1,7 +1,24 @@
 import React from "react";
+import Nav from "../../../../src/components/Nav/Nav";
+import User, { mainUser } from "./components/User";
+import Article from "../Main/components/Article";
+import Aside from "../Main/components/Aside";
+import "./Main.scss";
 
 function MainLee() {
-  return <h1>Hello world</h1>;
+  return (
+    <div>
```

Comment:
특정 스타일을 가지고 있지 않고, 단순 감싸기 위한 태그라면 Fragment 활용!
```suggestion
    <>
```